.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -3,15 +3,27 @@ name: 🎉 Feature request
 about: Suggest an idea to improve GitHub Desktop
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I have an issue when [...]
+## Goal
+A clear and concise description of what the problem and request are. Ex. I have an issue when [...problem...]. I'd like [...feature...] so that [...reason...]
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen. Add any considered drawbacks.
+#### Background
+Any related issues or any preceding features that this would depend on. (linked from desktop repository)

```

Comment:
A couple of :art: suggestions:

 - `preceding features that this might depend on` to allow for some flexibility of interpretation
 - `Links to other Desktop discussions are appreciated.` to encourage cross-referencing